yarn.lock

```diff
@@ -1071,6 +1071,13 @@
   version "2.0.0"
```

Comment:
since you didn't add any package, `yarn.lock` must not change

Reply:
Ok!

Reply:
@matgentili please check this.

Reply:
@fpersico Can i remove yarn.lock ? I don't add any package

Reply:
Just revert the changes.